go/engine/concurrency_test.go

```diff
@@ -126,7 +112,7 @@ func TestConcurrentGetPassphraseStream(t *testing.T) {
 }
 
 // TestConcurrentLogin tries calling logout, login, and many of
-// the exposed methods in LoginState concurrently.  Use the
+// the exposed methods in ActiveDevice concurrently.  Use the
```

Comment:
have you run with the `-conc` flag?  And `-race`

Reply:
nope, will do, can you give me the exact command to use?

Reply:
go test -conc -race -run Conc

Reply:
Pretty broken. I'll come back to it later...